saleor/graphql/product/descriptions.py

```diff
@@ -1,4 +1,5 @@
 class AttributeDescriptions:
+    INPUT_TYPE = "The type of rendering for this attribute and its values."
```

Comment:
I would slightly change it:

```suggestion
    INPUT_TYPE = "The input type to use for entering attribute values in the dashboard."
```